Signal/src/UIApplication+OWS.swift

```diff
@@ -9,8 +9,20 @@ extension UIApplication {
     var frontmostViewController: UIViewController? {
         let window = UIApplication.shared.keyWindow
         var viewController = window!.rootViewController
-        while viewController?.presentedViewController != nil {
-            viewController = viewController?.presentedViewController
+        
+        while true {
```

Comment:
This slightly changes the semantics of `frontmostViewController` by returning the top-most view in a `UINavigationController` rather than the `UINavigationController` itself.  

We mostly use `frontmostViewController` to present alerts and modals, in which case this change will have no effect.  This change however allows us to discriminate the current user-visible view in cases where we need to.